account_reset_chart/models/res_company.py

```diff
@@ -163,4 +152,30 @@ def unlink_from_company(model):
         unlink_from_company('account.tax.code')
         unlink_from_company('account.journal')
         unlink_from_company('account.account')
+
+        try:
+            self.env['account.partial.reconcile'].search(
+                [('company_id', '=', self.id)]
+            ).unlink()
+        except KeyError:
```

Comment:
This model is defined in the account module in 9.0, so this defense might not be needed.